StackExchange.Profiling.EntityFramework6/Properties/AssemblyInfo.cs

```diff
@@ -11,5 +11,5 @@
 [assembly: AssemblyCulture("")]
 [assembly: ComVisible(false)]
 [assembly: Guid("d56997c3-6d37-4953-bd49-b7f690390735")]
-[assembly: AssemblyVersion("3.0.11")]
-[assembly: AssemblyInformationalVersion("3.0.11")]
\ No newline at end of file
+[assembly: AssemblyVersion("3.1.0.0")]
+[assembly: AssemblyInformationalVersion("3.1.0.0")]
```

Comment:
I don't think this is necessary nor it is common practice as it seems (e.g. MiniProfiler's nuspec doesn't match with the assembly version). In general i found bumping up assembly version too cumbersome since i needed to go and update the versions in the config files